.github/workflows/ci.yml

```diff
@@ -39,4 +42,11 @@ jobs:
 
       - name: g8Test2.13
         if: startsWith(matrix.scala,'2.13.1')
-        run: sbt g8Test
+        run: | 
+          sbt g8Test
+          cd target/sbt-test/http4s-g8/scripted
+          sbt assembly
+          gu install native-image
+          cat native-image-readme.md | grep 'native-image  -H*' | sh
+          doalarm () { perl -e 'alarm shift; exec @ARGV' "$@"; }
+          doalarm 30 ./quickstartBinaryImage && exit 0
```

Comment:
Instead of sleeping for 30 seconds, maybe we could hit it with a curl (and even better test!), and if successful, kill the server?

Reply:
Sure, I was just excited to get a native binary built and launched via CI. Still need to figure out how to return an appropriate exit code so the build succeeds. Killing the binary returns an exit code of 143.

Reply:
Do you have some suggestions about elegant tests? I've just been trying to ship a working image build via CI quickly. Curl makes some sense and I think Jokes should be running so we could probably get a few jokes via curl or some other request engine. We could even try to script all the executions via sbt and launch tests via scala / https requests or otherwise. But maybe we should decompose this into seperate "stories". IE. publish a MVP now. Then make a define a testing issue and beef it up with well defined goals to boot.

Reply:
It might be good to have a brainstorming session via chat and then write up the issue.

Reply:
Yeah, incremental progress is good.  I'm fine with just confirming that the build works, and then we can aim higher by starting the server, curling, etc in another PR.

I don't have a lot of experience with sbt scripted and am not sure what the best way to test is, but we can discuss that elsewhere.